test/dynamo/test_misc.py

```diff
@@ -7630,10 +7630,10 @@ def my_dyn_fn(x):
 
         y = torch.randn([3])
         torch._dynamo.mark_static(y, 0)
-        torch._dynamo.optimize(counter)(my_dyn_fn)(y)
+        torch.compile(my_dyn_fn, backend=counter)(y)
 
         z = torch.randn([4])
-        torch._dynamo.optimize(counter)(my_dyn_fn)(z)
+        torch.compile(my_dyn_fn, backend=counter)(y)
```

Comment:
typo

Reply:
@soulitzer Thanks for your careful review! Will make changes and check all changes again.